test/driver-code.py

```diff
@@ -0,0 +1,77 @@
+import argparse
+import os
+
+class Dredd:
+    def __init__(self, endpoint, user, path, test_name):
+        if endpoint is not None:
+            self.endpoint = endpoint
+        else:     
+            self.endpoint = "https://127.0.0.1:9200"
+
+        if user is not None:
+            self.user = user
+        else:
+            self.user = "admin:admin"
+
+        if path is not None:
+            self.path = path
+        else:
+            self.path = ""    
+
+        if test_name is not None:
+            self.test_name  = test_name 
+        else:
+            self.test_name = ""  
+
+    def write_file(self): 
+        file_obj = open("url.txt", mode='w', encoding='utf-8') 
+        text = self.endpoint + " " + self.user 
+        file_obj.write(text)
+        file_obj.seek(0,0)
+        file_obj.close()
+
+    def dredd_work(self):
+        # Walking in test directory tree and runing dredd framework.
+        test_failed = 0
+        test_passed = 0
+        for dirpath, dirnames, files in os.walk("./models"+self.path):
+            curr_path = dirpath.split('/')
+            curr_dir = curr_path[len(curr_path)-1]         
+            if files:
+                command = "dredd " + dirpath +"/"+ files[1]+ " " + self.endpoint+ " --user=" + self.user + " --hookfiles=" + dirpath + "/" + files[0]
+                if self.test_name != "":
+                    if self.test_name == curr_dir:
+                        result = os.system(command)
+                        if(result != 0):
+                            test_failed = test_failed+1  
+                        else:
+                            test_passed = test_passed+1                                  
+                else:
+                    result = os.system(command)  
+                    if(result != 0):
+                        test_failed = test_failed+1
+                    else:
+                        test_passed = test_passed+1  
+        print("Total test cases: ", test_passed+test_failed)
+        print("Test failed: ", test_failed)
```

Comment:
Lets print the failed tests as form of table, Please check pretty-table

Reply:
Added a script that prints tables for failed test cases by default and generates tables for passed test cases with an additional argument (--testpass True).